bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/exceptions/EmptyResponseException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.exceptions;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Thrown when API is returning an unexpected empty response.
+ *
+ * @author Laurent Garnier - Initial contribution
+ */
+@SuppressWarnings("serial")
```

Comment:
I would suggest to remove this suppression and instead add:
```java
private static final long serialVersionUID = 1L;
```

Reply:
I copy/paste another exception of this binding which are all using @SuppressWarnings("serial"). 
I keep it on the new class to be consistent with others.